example/helloworld/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Dart gRPC sample client and server.
 homepage: https://github.com/dart-lang/grpc-dart
 
 environment:
-  sdk: '>=2.2.0 <3.0.0'
```

Comment:
Given the bump in minimum SDK requirements you should also bump the package version and add a CHANGELOG.md entry describing the change.

Reply:
> you should also bump the package version and add a CHANGELOG.md entry describing the change.
I see the CHANGELOG.md like this:
```
## 2.4.0

* Add the ability to bypass CORS preflight requests.

## 2.3.0

* Revert [PR #287](https://github.com/grpc/grpc-dart/pull/287), which allowed
using gRPC-web in native environments but also broke streaming.

## 2.2.0+1
...
```
Is it should append the new change when new version release?